decidim-core/lib/decidim/core.rb

```diff
@@ -880,6 +880,13 @@ def self.organization_settings(model)
     0.seconds
   end
 
+  # Disable Etiquette validator
+  # The etiquette validator is applied to the create forms of Proposals, Meetings,
+  # and Debates for both regular and admin users.
+  config_accessor :disable_etiquette_validator do
+    false
```

Comment:
It should be enable, to be consistent with other accessors (enable_html_header_snippets, enable_machine_translations, enable_remember_me, etc)
```suggestion
  config_accessor :enable_etiquette_validator do
    true
```